apps/authentication/mfa/__init__.py

```diff
@@ -1,5 +1,5 @@
+from .custom import MFACustom
+from .face import MFAFace
 from .otp import MFAOtp, otp_failed_msg
-from .sms import MFASms
 from .radius import MFARadius
-from .custom import MFACustom
-from .face import MFAFace
\ No newline at end of file
+from .sms import MFASms
```

Comment:
The code seems to be perfectly fine without any differences or issues identified in my knowledge cutoff of September 2021. Any changes might occur due to software updates or modifications that were beyond this point.

I suggest reviewing existing projects on GitHub, forums, and community discussions if you find yourself needing more detailed analysis than the current information provided by these checks offers.